src/jobs/test.yml

```diff
@@ -0,0 +1,52 @@
+parameters:
+  before-setup:
+    description: steps to execute before setup
+    type: steps
+    default: []
+  cache_key_prefix:
+    description: key of cache
+    type: string
+    default: ''
+  executor:
+    description: executor for test
+    type: executor
+  plugin_name:
+    description: name of Redmine plugin
+    type: string
+    default: ''
+  script:
+    description: steps to execute the test
+    type: steps
+    default:
+      - run: |
+          if [ -n "$REDMINE_PLUGIN_NAME" ]; then
+            bin/rails redmine:plugins:test NAME=$REDMINE_PLUGIN_NAME
```

Comment:
Since many Redmine plugins in the wild use minitest, this default value looks reasonable.

But in fact, most of Redmine plugins in agileware-jp org use RSpec. So when adopting this orb, perhaps we will want to switch test command automatically like this:

```sh
PLUGIN_NAME = ${REDMINE_PLUGIN_NAME:-$CIRCLE_PROJECT_REPONAME}
if [ -d plugins/$PLUGIN_NAME/spec ]; then
  bundle exec rspec plugins/$PLUGIN_NAME/spec
else
  bin/rails redmine:plugins:test NAME=$PLUGIN_NAME
fi
```

I'd like to leave a decision to our future self (or a next PR if you like :octocat: ).

Reply:
When the time comes, I'll add rspec job ✨💎✨:octocat: 

- test job for the test-unit/minitest user
- rspec job for the RSpec user
- turnip/cucumber job for the Gherkin user, maybe or maybe not